networking/change-block-size.md

```diff
@@ -19,9 +19,67 @@ This how-to guide uses the following {{site.prodname}} features:
 
 ### Concepts
 
+#### About IP pools
+
+The {{site.prodname}} IPAM block sizes for IP pools are:
```

Comment:
I think we should say something like this.

By default, Calico uses an IPAM block size of 64 addresses - /26 for IPv4, and /122 for IPv6. However, the block size can be changed depending on the IP pool address family.
- IPv4: 20-32, inclusive
- IPv6, 116-128, inclusive.

Reply:
Fixed.